src/ResourceManager/DevTestLabs/Commands.DevTestLabs/Extensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Linq;
+
+namespace Microsoft.Azure.Commands.DevTestLabs
+{
+    internal static class Extensions
+    {
+        internal static T DuckType<T>(this object target)
+        {
+            if (target == null)
+            {
+                return default(T);
+            }
+
+            return JsonConvert.DeserializeObject<T>(JsonConvert.SerializeObject(target));
```

Comment:
As per previous comment, this seems computationally inefficient.  Please use a proper type conversion.